news/2024/2024-09-17-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: Performance Points & Star Rating Updates
+date: 2024-09-17 12:00 +0000
+---
+
+The time for change is finally here!
+
+![](/wiki/shared/news/banners/pp-sr-osu.jpg)
+
+There are changes to all game modes this time around in an effort to improve the difficulty and performance calculations. This news post will display what has changed in a format that should be understandable to you, the player!
+
+If you find yourself scratching your head when reading, consider consulting the [performance points](/wiki/Performance_points) and [star rating](/wiki/Beatmapping/Star_rating) wiki articles to gain a greater understanding of the topics.
+
+## Release schedule
+
+<!-- todo -->
+
+## osu!catch performance points committee
+
+Since the last performance points post, osu!catch has now formed its own Performance Points Committee to help out with osu!catch related changes. You can find out more about the various committees on their [wiki page](/wiki/People/Performance_Points_Committee).
+
+---
+
+## osu! changes
+
+As a reminder of the various core values in the difficulty calculation algorithm:
+
+- **Aim**: how difficult it is to consistently hit notes back-to-back (affected by approach rate, jumps, Hidden, Hard Rock and more)
```

Comment:
Are these mentioned anywhere more permanently on the wiki? It feels like we want these definitions to be very front-and-foremost as not even *I* have an amazing concept of them.

Reply:
They are mentioned on https://osu.ppy.sh/wiki/en/Performance_points#calculation, however looking at it I notice it's slightly outdated (for example, Flashlight is no longer part of Aim calculation)